cmd/vic-machine/configure/configure.go

```diff
@@ -221,6 +231,14 @@ func (c *Configure) copyChangedConf(o *config.VirtualContainerHostConfigSpec, n
 	if n.RegistryCertificateAuthorities != nil {
 		o.RegistryCertificateAuthorities = n.RegistryCertificateAuthorities
 	}
+
+	if n.InsecureRegistries != nil {
+		o.InsecureRegistries = n.InsecureRegistries
+	}
+
+	if n.RegistryWhitelist != nil {
+		o.RegistryWhitelist = n.RegistryWhitelist
+	}
```

Comment:
oh @emlin it's already in the codebase :laughing: From what I can tell line 231 is what copies that across; at this point, after `Validate` has been called, `registryCAs` has been copied from `Data` to the new config, and it's held in `RegistryCertificateAuthorities`; I included that no-op apparently during my last PR